app/controllers/users/confirmations_controller.rb

```diff
@@ -27,6 +27,17 @@ def pending; end
 
 protected
 
+  def update_user
+    @confirmable.update_details(params[:user])
```

Comment:
Wonder if we changed the shape of the params, we could get organisation creation for free?

params = {
  user: {
    name: 'bob',
      organisation: {
         name: 'bestorg'
      }
  }
}

Reply:
Looks like you can but you have to pass an `Organisation` object, rather than a hash. So we still need the organisation creation step.

I got this working:
```
    organisation = @confirmable.build_organisation(name: params[:organisation_name])
    return true if organisation.save && user.save
```

But I don't know if its an improvement on the origin because we now have to call save on the user again. Before we didn't need to:

```
    organisation = Organisation.new(name: params[:organisation_name], users: [@confirmable])
    return true if organisation.save
```
Thoughts?